addons/l10n_be/models/template_be.py

```diff
@@ -79,19 +79,4 @@ def _get_be_reconcile_model(self):
                 'name@nl': 'Bankkosten (Geen BTW)',
                 'name@de': 'Bankgebühren (Ohne MwSt.)',
             },
-            'virements_internes_template': {
-                'name': 'Internal Transfers',
-                'to_check': False,
-                'line_ids': [
-                    Command.create({
-                        'account_id': 'a58',
```

Comment:
Don't we want to keep the same account on `internal_transfer_reco`)?

Reply:
Since the transfer account default to 'a58' for `l10n¸be` and the default transfer account is used by default for transfers, it does not change :smile: 

```40: 'transfer_account_id': 'a58', ```